src/index.js

```diff
@@ -1,18 +1,19 @@
 // @flow
 export type EqualityFn = (newArgs: mixed[], lastArgs: mixed[]) => boolean;
 
-const shallowEqual = (newValue: mixed, oldValue: mixed): boolean =>
-  newValue === oldValue;
-
-const simpleIsEqual: EqualityFn = (
-  newArgs: mixed[],
-  lastArgs: mixed[],
-): boolean =>
-  newArgs.length === lastArgs.length &&
-  newArgs.every(
-    (newArg: mixed, index: number): boolean =>
```

Comment:
no longer creating a new function on every call

Reply:
It's not a performance concern for V8, `every` could be even faster!
You need a benchmark to compare with.

Reply:
agreed